BrainPortal/public/javascripts/userfiles.js

```diff
@@ -773,15 +793,6 @@ $(function() {
             $('#up-type').val(data).trigger('change.uf');
           });
         })
```

Comment:
It's still there, I just moved everything that modified the "disabled" status of the upload button into one method so they don't undo the work of the other ones